packages/compass-e2e-tests/helpers/commands/connect-with-connection-form.js

```diff
@@ -61,62 +61,62 @@ module.exports = function (app) {
         ? 'MONGODB'
         : 'NONE';
 
-    const authStrategyInputComponent = await client.$(
+    const authStrategyInputComponent = await browser.$(
       Selectors.ConnectionFormInputAuthStrategy
     );
     await authStrategyInputComponent.selectByAttribute('value', authStrategy);
 
     if (typeof username !== 'undefined') {
-      const kerberosPrincipalInputElement = await app.client.$(
+      const kerberosPrincipalInputElement = await compass.browser.$(
```

Comment:
for consistency with other parts of the code here

```suggestion
      const kerberosPrincipalInputElement = await browser.$(
```